src/pybind/mgr/dashboard/frontend/src/app/shared/components/components.module.ts

```diff
@@ -88,6 +89,8 @@ import { UsageBarComponent } from './usage-bar/usage-bar.component';
     OrchestratorDocPanelComponent,
     DateTimePickerComponent,
     DocComponent,
+    Copy2ClipboardButtonComponent,
+    DocComponent,
```

Comment:
`DocComponent` is duplicated

Reply:
Should be resolved in the most recent push! Looks like it got pulled in when resolving the merge conflicts (`DocComponent` was the last in the list, and needed a comma added, so both versions modified it.)